sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -293,6 +293,13 @@ class SQLContext(@transient val sparkContext: SparkContext)
   }
 
   /**
+   * Returns the list of tables for this SQLContext
+   */
+  def showTables(): Seq[String] = {
```

Comment:
@marmbrus if we want to make showTables work for both Java and Scala, maybe this can return an Array?